datascience/tables.py

```diff
@@ -2306,14 +2307,18 @@ def scatter(self, column_for_x, select=None, overlay=True, fit_line=False,
                 for additional arguments that can be passed into vargs. These
                 include: `marker` and `norm`, to name a couple.
 
-            ``colors``: A column of categories to be used for coloring dots.
+            ``group``: A column of categories to be used for coloring dots per 
+                each category grouping.
 
             ``labels``: A column of text labels to annotate dots.
 
             ``sizes``:  A column of values to set the relative areas of dots.
 
             ``s``: Size of dots. If sizes is also provided, then dots will be
-              in the range 0 to 2 * s.
+                in the range 0 to 2 * s.
+
+            ``colors``: (deprecated) A synonym for ``group``.  Retained
```

Comment:
Let's say "Retained temporarily for backwards compatibility. This argument will be removed in future releases.".

Reply:
Oh, I like that.  Done.